core/templates/pages/contributor-dashboard-page/modal-templates/login-required-modal.component.spec.ts

```diff
@@ -26,18 +26,17 @@ describe('Login Required Modal Content', () => {
   let component: LoginRequiredModalContent;
   let fixture: ComponentFixture<LoginRequiredModalContent>;
 
-  beforeEach(() => {
+  beforeEach(waitForAsync(() => {
```

Comment:
Why any changes in this file?

Reply:
tests were failing without these changes.